package.json

```diff
@@ -87,31 +21,140 @@
       ],
       "items": {
         "type": "string"
+      },
+      "order": 1
+    },
+    "lintHtmlFiles": {
+      "title": "Lint HTML Files",
+      "description": "You should also add `eslint-plugin-html` to your .eslintrc plugins",
+      "type": "boolean",
+      "default": false,
+      "order": 2
+    },
+    "autofix": {
+      "type": "object",
+      "order": 3,
+      "title": "Automatic Fixes",
+      "properties": {
+        "fixOnSave": {
+          "title": "Fix errors on save",
+          "description": "Have eslint attempt to fix some errors automatically when saving the file.",
+          "type": "boolean",
+          "default": false,
+          "order": 1
+        },
+        "ignoreFixableRulesWhileTyping": {
+          "title": "Ignore fixable rules while typing",
+          "description": "Have the linter ignore all fixable rules during linting when editing a document. The list is automatically updated on each lint job, and requires at least one run to be populated. Only supported when using ESLint v4+.",
+          "type": "boolean",
+          "default": false,
+          "order": 2
+        },
+        "rulesToDisableWhileFixing": {
+          "title": "Disable specific rules from fixes",
+          "description": "Prevent rules from being auto-fixed by ESLint. Applies to fixes made during saves as well as when running the `Linter Eslint: Fix File` command.",
+          "type": "array",
+          "default": [],
+          "items": {
+            "type": "string"
+          },
+          "order": 3
+        }
       }
     },
-    "rulesToSilenceWhileTyping": {
-      "title": "Silence specific rules while typing",
-      "description": "Useful when Atom fixes errors on save like `no-trailing-spaces` or `eol-last`.",
-      "type": "array",
-      "default": [],
-      "items": {
-        "type": "string"
+    "global": {
+      "type": "object",
+      "order": 4,
+      "title": "Global ESLint",
+      "properties": {
+        "useGlobalEslint": {
+          "title": "Use global ESLint installation",
+          "description": "Make sure you have it in your $PATH",
```

Comment:
Could we add something here to emphasize that local installation is the suggested usage?

Reply:
As I mentioned to @Arcanemagus in one of the previous comments, I'd like to save wording tweaks for another PR.  There is a lot that I would like to do to help explain and clarify our settings, but this PR is really just about reorganizing them.

Reply:
Fair enough. 👍